apps/e2e-tests/config.ts

```diff
@@ -0,0 +1,4 @@
+export const config = {
+  appOrigin: process.env.APP_ORIGIN || 'https://api-dev.hub.asap.science',
```

Comment:
I think app origin so far usually refers to the frontend, we probably have some other environment variables in use for the api origin?